lib/Mad/Mapper.pm

```diff
@@ -253,16 +255,26 @@ sub import {
   }
 
   if ($flag) {
+    my @attr = @_;
     my $caller = caller;
     my $table = Mojo::Util::decamelize((split /::/, $caller)[-1]);
-    $table =~ s!s?$!s!;    # user => users
+    if ( grep { $_ eq '-proper_plural' } @attr ) {
```

Comment:
Let's keep it simple: Just use Lingua::EN::Inflect in all cases. Also add Lingua::EN::Inflect to "cpanfile".